test/styles/RRR.R

```diff
@@ -39,38 +48,110 @@ fun_call(argument1,
 fun_call(
     argument1,
     argument2
-    )
+)
 
 ## 3
 fun_call(parameter = (
     stuff
-    ),
-         argument)
+),
+argument)
 
 ## 4
 fun_call(parameter = fun_argument(
-             stuff
-    ),
-         argument)
+    argument1
+),
+argument2)
+
+## 5
+fun_call(parameter = fun_argument(argument1,
+                                  argument2
+                                  )
+        ,
+         argument3)
```

Comment:
> To get the last behaviour you set ess-offset-arguments-newline to t instead of '(t). Should it be the default for RRR?

But does this affect indentation of `{`?